common/components/time/time.js

```diff
@@ -0,0 +1,74 @@
+const { differenceInMinutes, isPast, parseISO } = require('date-fns')
```

Comment:
There's a few lines of complicated code here - do we have any easy way to test this at the moment?

Reply:
I was struggling a little with it to be honest. I don't think the helpers that load the components also load in the javascript...I'll have a think how we could do it...